Feedback dialog ui

nuddin175 opened the pull request:
- Added feedback modal to interviews. Currently accessible through /interview route with a default interivewID. Will change later to contain id from params
- created a backend route to get feedback based on the creator.

![image](https://user-images.githubusercontent.com/61724322/100483751-f4e1dc80-30c7-11eb-8f7b-56af64388699.png)
![image](https://user-images.githubusercontent.com/61724322/100483764-032ff880-30c8-11eb-8b3e-0fbd837eb6b7.png)
![image](https://user-images.githubusercontent.com/61724322/100483784-1773f580-30c8-11eb-8d18-9a2f46b96aa9.png)
![image](https://user-images.githubusercontent.com/61724322/100483813-235fb780-30c8-11eb-90cf-7a7660d6d01c.png)

nuddin175 commented:
- Updated interviewId to be retrieved from params

ThinkTink commented on client/src/utils/apiFunctions.js:
```diff
@@ -37,9 +37,21 @@ export const logoutUser = () => {
+};
+
+export const getFeedbackCreator = (interviewId) => {
+  return api.get(`/interviews/${interviewId}/feedback/creator`);
```
Is creator in the function name and route because it's the feedback you've created vs the feedback you've received?

ThinkTink reviewed (commented):
Needs just a bit of refactoring so that we don't make the same API call 7 times

nuddin175 commented:
- Refactored the feedback dialog to call api in the top component
- Added past interview table showing feedback scores, and linking to full feedback and questions
